### ISSUE TITLE
h2 mix data from different table rows when accessing blob fields from different threads in java application

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#284]
```
What steps will reproduce the problem?
(simple SQL scripts or simple standalone applications are preferred)
Standalone application attached, see README.txt for details

What is the expected output? What do you see instead?
What version of the product are you using? On what operating system, file
system, and virtual machine?
- latest stable h2 (attached), occurs also in case of earlier versions
- java version "1.6.0_20"
- op system: linux
- file system: issue occurs both in local file system and in nfs, but more 
frequent in case of nfs.

Do you know a workaround?
No workaround known.

How important/urgent is the problem for you?
Very important.

In your view, is this a defect or a feature request?
Defect.

Please provide any additional information below.
See attachment
```

Original issue reported on code.google.com by `drucza.m...@gmail.com` on 4 Feb 2011 at 3:19

**Comment by GoogleCodeExporter** [comment]
```
Hi,

I can reproduce the problem now. A workaround is to append ;QUERY_CACHE_SIZE=0 
to the database URL, even then the test fails. I will fix this for the next 
release.

Thanks a lot for your help!

Regards,
Thomas

```

Original comment by `thomas.t...@gmail.com` on 9 Feb 2011 at 7:40
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Hi Thomas,

Thank you for the reply!
I' m looking forward to see the fix in next release.

Regards,
Miklos
```

Original comment by `drucza.m...@gmail.com` on 10 Feb 2011 at 9:04

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.3.151.
```

Original comment by `thomas.t...@gmail.com` on 12 Feb 2011 at 2:46
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Hi Thomas,

Thank you for fixing the issue. It' s working correctly on my computer also.
When do you plan to publish the stable release of 1.3.151 version?

Thanks,
Miklos
```

Original comment by `drucza.m...@gmail.com` on 14 Feb 2011 at 3:12
